apitools/org.eclipse.pde.api.tools/src_ant/org/eclipse/pde/api/tools/internal/tasks/APIToolsAnalysisTask.java

```diff
@@ -175,9 +175,7 @@ private Summary[] createAllSummaries(Map<String, IApiProblem[]> allProblems) {
 		}
 		List<Entry<String, IApiProblem[]>> allEntries = new ArrayList<>();
 		allEntries.addAll(entrySet);
-		Collections.sort(allEntries, (entry1, entry2) -> {
-			return entry1.getKey().compareTo(entry2.getKey());
-		});
+		Collections.sort(allEntries, (entry1, entry2) -> entry1.getKey().compareTo(entry2.getKey()));
```

Comment:
In a follow-up a static comparator factory could be used here and at other occasions:

```suggestion
		Collections.sort(allEntries, Comparator.comparing(Entry::geKey));
```